src/search/indexing/local_import.rs

```diff
@@ -53,23 +53,23 @@ pub async fn index_local(pool: PgPool) -> Result<Vec<SearchMod>, IndexingError>
                 icon_url = url;
             }
 
-            docs_to_add.push(SearchMod {
-                mod_id: result.id,
+            docs_to_add.push(UploadSearchMod {
+                mod_id: format!("local-{}", crate::models::ids::ModId(result.id as u64)),
                 author: "".to_string(),
                 title: result.title,
                 description: result.description,
-                keywords: categories,
+                categories,
                 versions,
                 downloads: result.downloads,
                 page_url: result.body_url,
                 icon_url,
                 author_url: "".to_string(),
                 date_created: result.published.to_string(),
-                created: 0,
+                created_timestamp: 0,
                 date_modified: "".to_string(),
-                updated: 0,
+                modified_timestamp: 0,
                 latest_version: "".to_string(),
-                empty: String::from("{}{}{}"),
+                empty: std::borrow::Cow::Borrowed("{}{}{}"),
```

Comment:
Might be out of scope of this PR, but we should fill in these values with proper stuff since a lot of the database stuff has been done.

Reply:
The only things that could change here without much work would be adding the `created_timestamp` as `result.published.timestamp()`.  We don't currently store the last modified time in the database for mods, we have no direct connection between mods and users for the author - we may be able to replace this with teams, and we don't currently have a system for tracking the latest version (This will likely require a separate PR, because it is complicated and is going to be hard to get right.)

Technically we could return the update time and timestamp from the created time and timestamp, but at some point we will need to change that.

Reply:
For now I've added the modified times as the created time, but the author/latest version fields are still blank.  I'll leave the rest for a later PR.